src/utils/Evaluator.js

```diff
@@ -74,6 +74,9 @@ const Evaluator = {
     }
     if (typeof template === 'function') {
       try {
+        if (data.component.filter === rawTemplate && !data.options.building) {
```

Comment:
The evaluator should have not knowledge of a form component and these kinds of options. I believe this change needs to be made lower in the call stack.

Reply:
moved to the interpolate method of Element.js